modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -340,3 +337,35 @@ Include Password Field:: Shows the password field.
 Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
+
+== Filter Conditions [[filter-conditions]]
+
+The API Carousel component and the API Catalog component use the parameter *Filter Condition (Advanced)* to show only the APIs that meet the condition. A condition is a standard search in Salesforce Object Query Language (SOQL).
+
+You can search these fields:
+
+* `+acm_pkg__Icon__c+`
+* `+acm_pkg__Name__c+`
+* `+acm_pkg__NavigateUrl__c+`
+* `+acm_pkg__VersionGroup__c+`
+
+These examples demonstrate filter conditions:
+
```

Comment:
This is quite a few examples! Do we need so many?
Do you suppose it would make more sense to show the condition first and then the meaning?

Reply:
True, 12 is a lot. However, I think the functionality each one shows is different enough from all the others to be worth including.

I like your idea of showing condition and then meaning. Fixed.